lldb/include/lldb/Utility/XcodeSDKPath.h

```diff
@@ -0,0 +1,48 @@
+//===-- XcodeSDK.h ----------------------------------------------*- C++ -*-===//
```

Comment:
I think I would just put this into the existing XcodeSDK.h header, since it's so closely related.